projects/hslayers/src/components/compositions/layer-parser/layer-parser.service.ts

```diff
@@ -227,7 +196,27 @@ export class HsCompositionsLayerParserService {
    * @param lyr_def - Layer definition object
    * @returns Ol Image or Tile layer
    */
-  createXYZLayer(lyr_def) {
+  async createXYZLayer(lyr_def) {
+    lyr_def.url = decodeURIComponent(lyr_def.url);
+    /**
+     * FIXME : NOT 100% reliable need to be revised along with
```

Comment:
Is this FIXME still relevant @FilipLeitner ? If so, could you create a separate issue for it?